linked_list/linked_list.py

```diff
@@ -23,13 +23,39 @@ def __init__(self):
     self.tail = None
 
   def add_to_tail(self, value):
-    pass
+    if self.tail is None:
+      first_node = Node(value)
+      self.head = first_node
+      self.tail = first_node
+    else:
+      newest_node = Node(value)
+      self.tail.next_node = newest_node
+      self.tail = newest_node
+      self.tail.next_node = None
 
   def remove_head(self):
```

Comment:
This is passing the tests. But because you are setting tail to `None` every single time you remove from head, you are breaking the ability to link any new nodes to your current list.  You actually only want tail to be None if removing the head causes the list to be empty.